yaptide/routes/batch_routes.py

```diff
@@ -79,14 +79,15 @@ def post(user: KeycloakUserModel):
         input_dict_to_save["number_of_all_primaries"] = number_of_all_primaries
         input_dict_to_save["input_files"] = files_dict
 
-        result = submit_job(payload_dict=payload_dict, files_dict=files_dict, user=user, cluster=cluster)
+        result = submit_job(payload_dict=payload_dict, files_dict=files_dict, user=user,
```

Comment:
At this point I'd extract backend URL from an environment variable. If not set I'd say it should go to localhost. Then this should go into the submit script.

Reply:
Added in lowest possible point